lib/string.go

```diff
@@ -0,0 +1,23 @@
+package lib
+
+import (
+	"strings"
+)
+
+// StrContains checks if a list contains a string
+func StrContains(l []string, s string) bool {
+	for _, v := range l {
+		if v == s {
+			return true
+		}
+	}
+	return false
+}
+
+func ToLowerList(l []string) []string {
```

Comment:
Ah sorry - I meant `ToLowerList` - was pointing at that line with my comment, but GitHub does a lousy job of showing that.